src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -168,6 +183,8 @@ export default class ResultsViewOncoprint extends React.Component<IResultsViewOn
         this.onMouseEnter = this.onMouseEnter.bind(this);
         this.onMouseLeave = this.onMouseLeave.bind(this);
 
+        // update URL parameters according to UI events
```

Comment:
we want to try to maintain the url as the main source of truth, so instead of setting these on the store, we would change the url and then let the app respond to that.

Reply:
@alisman I think you are saying we should no longer have a JS representation for columnMode, heatmapTrackGroupsUrlParam, and clinicalTracksUrlParam - any code that tries to update those should be rewritten to update straight into the URL. That sounds like a good idea, but unrelated to this PR.

Reply:
Does that mean that we can get this merged now? @adamabeshouse @alisman

Reply:
@Sjoerd-van-Hagen we still don't understand the issue of refreshing, these reactions should be combined. @pvannierop

Reply:
Sure, no problem. How high is this on your list of priorities? We would like to see this in the next release if possible.

Reply:
@Sjoerd-van-Hagen sorry, to be clear, we need @pvannierop 's clarification to our confusion please.

I'm not sure where it falls on our priorities but I think it wouldn't be too problematic to merge @alisman

Reply:
Right, you need a reply to the other comment you made.

Pim is OOO until Wednesday though :-(

Reply:
Can we wait until then?

Reply:
Yes, let's do that then. I was actually under the impression that Pim has had a call with @alisman about this.

Reply:
Yes, I had a call with @alisman. I was under the impression that @adamabeshouse and @alisman were internally discussing a solution to the refresh problem.

The refresh of oncoprint refers to the loading icon appearing when triggering `updateRoute()` command after setting the `oncoprint_sortby` url param. This behavior is different from toggling the sort order via the menu (no refresh of oncoprint) and is, therefore, undesired.

I propose this include this topic in our telco of this afternoon.